tasks/fh-build.js

```diff
@@ -403,12 +387,10 @@ module.exports = function(grunt) {
         'shell:fh-run-array:integrate_cover', 'shell:fh-run-array:accept_cover',
         'shell:fh-report-cov:lcov', 'shell:fh-report-cov:cobertura'
       ]);
-    } else if (this.target === 'analysis') {
-      grunt.task.run(['plato:fh']);
```

Comment:
This is going to break some builds. Example:
https://github.com/fheng/jenkins-bob-builder/blob/d71c8874c9bcaf64052fab491aaa28bbf9392da4/jobs/node-pullrequests/projects.yaml#L74